src/hooks/useTheme/useTheme.tsx

```diff
@@ -0,0 +1,11 @@
+import { useLayoutEffect, useState } from "react";
+
+export const useTheme=()=>{
+    const [theme,setTheme]=useState(localStorage.getItem("theme") || "dark")
```

Comment:
instaed of 'dark' use the const from previous file that corresponds dark theme:

import {THEME} from ...

const defaultTheme = THEME.DARK
const [theme, setTheme] = useState(localStorage.getItem("theme") || defaultTheme)